rootfs/deploy.py

```diff
@@ -31,7 +31,8 @@ def log_output(stream, decode):
         s3 = boto3.client('s3', config=Config(signature_version='s3v4'),
                           endpoint_url=hosturl,
                           aws_access_key_id=secretId,
-                          aws_secret_access_key=secretKey)
+                          aws_secret_access_key=secretKey,
+                          region_name=os.environ.get('REGION', 'us-east-1'))
```

Comment:
Let's keep it simple for now. If I have to change this I have to make appropriate changes in builder too.